python/lsst/ts/criopy/m1m3/force_actuator/HistogramPageWidget.py

```diff
@@ -94,7 +94,7 @@ def updateValues(self, data: typing.Any) -> None:
         data : `object`
             New data structure, passed from SAL handler.
         """
-        if data is None:
+        if data is None or self.field is None:
```

Comment:
I thought `black` will do?

```
if (data is None) or (self.field is None):
```

Reply:
No. It passes pre-commit, black doesn't do that. Even when running black on the file, this isn't changed.